addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/automation.html

```diff
@@ -9,15 +9,51 @@ <H1>Scripts Automation Framework Support</H1>
 	This add-on supports the Automation Framework.
 
 	<H2>Job: script</H2>
-    The script job allows you to execute various actions with scripts.
-    Run standalone scripts is currently the only action, but more actions are planned (add/remove/enable/disable)
+    The script job allows you to execute various actions with scripts:
+    
+	<H2>Action: add</H2>
+
+	Adds the specified script to ZAP. Scripts are enabled but not configured to be loaded when ZAP restarts.
+	<br><br>
+	By default the default script engine for the file extension (if any) will be used - this may be overridden using the 'engine' parameter.
+	
+	<ul>
+	<li>type: mandatory, can be any of the script types supported by ZAP
+	<li>engine: optional, can be used to override the default engine for the file extension
+	<li>name: optional, defaults to the file name, can be used to specify the script in another job
+	<li>file: mandatory, the full file path, must be a readable text file
+	</ul>
+	
+	<H2>Action: remove</H2>
+
+	Removes the specified script from ZAP.
+	
+	<ul>
+	<li>type: mandatory, can be any of the script types supported by ZAP
```

Comment:
Is this really needed? The name alone can be used to uniquely identify the script, it's odd that it also requires a script type.